src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+
+    private double radius;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        int exponent = 2;
+        return Math.PI * Math.pow(radius, exponent);
+    }
+
+    @Override
+    public String info() {
+        setFigureName("circle");
+        setRadius(random.nextDouble() + 1);
+        setColor(colorSupplier.getRandomColor());

```

Comment:
you don’t need set values in this method. this method must return info about the model and that’s  all